README.md

```diff
@@ -118,6 +118,31 @@ This helps to avoid unnecessary effort by you and us. Use the issue system
 or [mail us](mailto:kalibrierzentrum@imf.tu-freiberg.de). If you want to create your own plugin package, please use
 the [plugin template](https://github.com/pyroll-project/pyroll-plugin-template) and follow the instructions there.
 
+### Policy for Inclusion of Hooks in the Core
+
+Since version 2.0 we try to include as many hooks as possible into the core, to tie down the nomenclature.
+This seems necessary, since for the same concepts, often different terms exist in literature.
+To avoid collisions and redundancies, it is better to include these hooks in the core, although some of them may come without implementation.
+This improves the exchangeability of plugins.
+If you like a hook to be included in the core, open an issue, or preferably, a pull request with the respective changes.
+Supply your pull request with fallback implementations of the hook, if any meaningful exist.
```

Comment:
I though that we don't want to have fallbacks in the core any more as the core is more like the calculative skeleton handling iterations, chacheing etc.

Reply:
Fallbacks mean, that they are basic physical relations. Complex or model dependent things are excluded. Case decision in the end.